### PR TITLE
Write-up of merged RFCs 673 & 674

### PR DESCRIPTION
**Pull request by ijlee2**

### CODE REVIEW

File: source/2020-11-06-the-ember-times-issue-167.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 New Release and APIs for ember-a11y-testing 🎉,
 New Octane ready release of the a11y focused ember-select-light 🎉,
+2 deprecation RFCs merged,
```
Comment (ijlee2) [inline source/2020-11-06-the-ember-times-issue-167.md:12]: I didn't have time to find an emoji. Feel free to add one.

File: source/2020-11-06-the-ember-times-issue-167.md [hunk 2]
```diff
@@ -116,7 +114,7 @@ Read more about the new changes to [ember-select-light on GitHub](https://github
...
...lank">Chris Manson (@mansona)</a>, <a href="https://github.com/JamesS-M" rel="noopener noreferrer" target="_blank">@JamesS-M</a>, <a href="https://github.com/bendemboski" rel="noopener noreferrer" target="_blank">Ben Demboski (@bendemboski)</a>, <a href="https://github.com/tylerturdenpants" rel="noopener noreferrer" target="_blank">Ryan Mark (@tylerturdenpants)</a>, <a href="https://github.com/scalvert" rel="noopener noreferrer" target="_blank">Steve Calvert (@scalvert)</a>, <a href="https://github.com/jbryson3" rel="noopener noreferrer" target="_blank">John Bryson (@jbryson3)</a>, <a href="https://github.com/chrisrng" rel="noopener noreferrer" target="_blank">Chris Ng (@chrisrng)</a>, <a href="https://github.com/drewlee" rel="noopener noreferrer" target="_blank">Andrew A Lee (@drewlee)</a>, <a href="https://github.com/kategengler" rel="noopener noreferrer" target="_blank">Katie Gengler (@kategengler)</a>, <a href="https://github.com/rwjblue" rel="noopener noreferrer" target="_blank">Robert Jackson (@rwjblue)</a>, and <a href="https://github.com/ijlee2" rel="noopener noreferrer" target="_blank">Isaac Lee (@ijlee2)</a> for their contributions to Ember and related repositories! 💖</p>
```
Comment (ijlee2) [inline source/2020-11-06-the-ember-times-issue-167.md:117]: Contributors will appear in a random order now.

File: source/2020-11-06-the-ember-times-issue-167.md (hunk 1 above)
Comment (amyrlam) [reply source/2020-11-06-the-ember-times-issue-167.md:12]: will do, thank you!

File: source/2020-11-06-the-ember-times-issue-167.md (hunk 2 above)
Comment (amyrlam) [reply source/2020-11-06-the-ember-times-issue-167.md:117]: that's rad! whoa, you are last :magic:

**Review by amyrlam — APPROVED** [review]
thanks @ijlee2! i'll take make some adds upstream

**Pull request merged**